deployment/build-s3-dist.sh

```diff
@@ -65,4 +66,10 @@ echo "--------------------------------------------------------------------------
 echo "[Packing] Custom Resource"
 echo "------------------------------------------------------------------------------"
 cd ../custom-resource
-zip -q -r9 ../../deployment/dist/v4/custom-resource.zip *
\ No newline at end of file
+zip -q -r9 ../../deployment/dist/v4/custom-resource.zip *
```

Comment:
`-r9` ?

Reply:
Not sure! It's in the original aws code, and it works fine, so I used the same syntax for my new folder. Can remove the 9 for my line, but will leave theirs, since trying to keep changes to the aws code to a minimum...